contrib/tracking_test.go

```diff
@@ -0,0 +1,65 @@
+package contrib
+
+import (
+	"gocv.io/x/gocv"
+	"image"
+	"testing"
+)
+
+func BaseTestTracker(t *testing.T, tracker Tracker, name string) {
+	if tracker == nil {
+		t.Error("TestTracker " + name + " should not be nil")
+	}
+	img := gocv.IMRead("../images/face.jpg", 1)
+	if img.Empty() {
+		t.Error("TestTracker " + name + " input img failed to load")
+	}
+	rect := image.Rect(250, 150, 250+200, 150+250)
+	init := tracker.Init(img, rect)
```

Comment:
maybe we should assert on the bounding box values to make sure we do the math correctly.

Reply:
the img is 640x480